scalafix-rules/src/main/scala/scalafix/internal/rule/ProcedureSyntax.scala

```diff
@@ -23,6 +27,21 @@ class ProcedureSyntax extends SyntacticRule("ProcedureSyntax") {
           toAdd <- doc.tokenList.leading(bodyStart).find(!_.is[Trivia])
         } yield Patch.addRight(toAdd, s": Unit =").atomic
         fixed.getOrElse(Patch.empty)
+
+      /** @see [[https://github.com/ohze/scala-rewrites/blob/dotty/rewrites/src/main/scala/fix/scala213/ConstructorProcedureSyntax.scala ConstructorProcedureSyntax.scala]] */
```

Comment:
@giabao it seems that https://github.com/ohze/scala-rewrites is not licensed, can you advise since this PR is basically inlining code from there (with the test) ?